doc/operators/container.rst

```diff
@@ -0,0 +1,23 @@
+Runtime Container
+=================
+
+With every release, a sample runtime base container is provided. It contains the ``cchost`` executable and its dependencies,
```

Comment:
we should probably say for "minor release", i.e. we might provide a containers for release candidates or for some minor release (depending on need)

Reply:
I’m not sure I understand your point. I would think that we provide containers for anything that is on the Release page of GitHub, regardless of how minor it is?

Reply:
@ashamis ?

Reply:
sure, just wanted to make sure this was part of the plan